.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: ruby
 rvm:
  - 2.5.3
+before_install: # somehow travis isn't coming with bundler preinstalled any more?!
+  - gem install bundler -v '>= 2' || true
```

Comment:
Later versions of Ruby than 2.5.3 come with `bundler` pre-installed, IIRC. That might be why?

Reply:
🤷‍♂ probably time to upgrade soon